Gemfile.lock

```diff
@@ -1,24 +1,25 @@
 GIT
   remote: https://github.com/projecttacoma/simplexml_parser.git
-  revision: 0ea6c4cf9c5cb70c848ee7beaf30779b9ea77f15
+  revision: 9f83211e2407f0d933afbd1648c57f500b7527af
   branch: master
   specs:
-    simplexml_parser (1.0.0)
+    simplexml_parser (1.1.0)
+      health-data-standards (~> 4.3.4)
```

Comment:
i'm not sure we need this change. we are using our fork of health data standard from pophealth but i'm not sure how would we avoid it. i guess we might have to leave it and see

Reply:
yeah im inclined to leave this change unless you feel strongly otherwise